outputs.tf

```diff
@@ -30,6 +30,21 @@ output "private_subnets_cidr_blocks" {
   value       = ["${aws_subnet.private.*.cidr_block}"]
 }
 
+output "ipv6_enabled_private_subnets" {
+  description = "List of IDs of private subnets in an ipv6 enabled VPC"
+  value       = ["${aws_subnet.private_ipv6.*.id}"]
+}
+
+output "ipv6_enabled_private_subnets_ipv4_cidr_blocks" {
+  description = "Map of List of ipv4 cidr_blocks of private subnets in an ipv6 enabled VPC"
```

Comment:
The description is a dangerous nice mix of ipv4 and ipv6 even if it makes sense :)

Reply:
Also, this is not `map of list`, but just a `list`

Reply:
Fixed in latest commit